sda-commons-client-jersey-example/src/test/java/org/sdase/commons/client/jersey/JerseyClientExampleIT.java

```diff
@@ -50,11 +45,6 @@
   private static final Car LIGHT_BLUE_CAR =
       new Car().setSign("HH XY 4321").setColor("light blue"); // NOSONAR
 
-  @BeforeClass
-  public static void start() {
-    WIRE.start();
-  }
```

Comment:
This is also a pattern that I see a lot. As a explanation why we should delete it: If we use Wiremock as a rule (with `@Rule` or via the `RuleChain`), it is automatically started and stopped. There is no need to do it manually.

Reply:
👍